src/category_theory/monoidal/category.lean

```diff
@@ -69,6 +69,7 @@ class monoidal_category (C : Type u) [𝒞 : category.{v} C] :=
 restate_axiom monoidal_category.tensor_id'
 attribute [simp] monoidal_category.tensor_id
 restate_axiom monoidal_category.tensor_comp'
+attribute [reassoc] monoidal_category.tensor_comp
 attribute [simp] monoidal_category.tensor_comp
```

Comment:
Do you want this to be `attribute [simp, reassoc] monoidal_category.tensor_comp`?

Reply:
I'll add a comment. The reassoc lemma would be redundant in the simp set, so the linter complains.